examples/with-sdk-server/tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "sourceMap": true,
     "target": "es2022",
     "types": ["node"],
-    "outDir": "dist"
+    "outDir": "dist",
+    "skipLibCheck": true
```

Comment:
any context on this addition?

Reply:
all of a sudden it was throwing the most obscure type error from the webauthn-p256 package:
```
> @turnkey/example-with-sdk-server@0.1.0 typecheck $HOME/code/src/github.com/tkhq/sdk/examples/with-sdk-server
> tsc --noEmit

../../node_modules/.pnpm/webauthn-p256@0.0.10/node_modules/webauthn-p256/_types/types.d.ts(27,67): error TS2536: Type '_K' cannot be used to index type 'fallback'.
 ELIFECYCLE  Command failed with exit code 2.
 ```
 
 so added `skipLibCheck` but open to an alternative solution if you have a suggestiong